Angular/package.json

```
@@ -6,7 +6,11 @@
     "start": "ng serve",
     "build": "ng build",
     "watch": "ng build --watch --configuration development",
-    "test": "ng test"
+    "test": "ng test",
+    "lint-html": "prettier --check .",
+    "lint-ts": "eslint --ext .ts .",
+    "lint-scss": "stylelint **/*.scss",
+    "lint": "npm-run-all -p -c lint-ts lint-scss lint-html"
```

minor: let's just 

```suggestion
    "lint": "lint-ts && lint-scss && lint-html"
```

and remove `npm-run-all` dependency

---

`"lint": "lint-ts && lint-scss && lint-html"`

with this code, running `npm run lint` will throw an error: "lint-ts is not recognized as an internal or external command..."

did you mean?: `"lint": "npm run lint-ts && npm run lint-scss && npm run lint-html"`
this works, but the commands will be executed in sequence and will not continue executing subsequent commands if there's an error compared to `npm-run-all -p -c`

---

thanks, now I see the difference